user_guide_src/source/incoming/controllers.rst

```diff
@@ -58,9 +65,9 @@ This is valid::
 
     <?php namespace App\Controllers;
 
-    use CodeIgniter\Controller;
+    use CodeIgniter\BaseController;
```

Comment:
Sorry, I probably should mention that all occurrences of this are not needed.

Also, there is no `BaseController` in the `CodeIgniter` namespace :)